uaa/build.gradle

```diff
@@ -60,6 +60,10 @@ dependencies {
     //implementation(libraries.dynamicLogConfigurer)
     runtimeOnly(libraries.nurego)
 
+    implementation(libraries.gevLogFilter) {
+        exclude(module: "slf4j-log4j12")
+    }
```

Comment:
16:15:53  > Could not resolve all files for configuration ':cloudfoundry-identity-uaa:compileClasspath'.
16:15:53     > Could not find com.ge.predix.gev-log-filter:1.0.0:.
16:15:53       Required by:
16:15:53           project :cloudfoundry-identity-uaa

not in dig ?